applications/nrf5340_audio/doc/index.rst

```diff
@@ -17,6 +17,9 @@ The nRF5340 Audio application demonstrates audio playback over isochronous chann
 .. note::
    This application and its DFU/FOTA functionality are marked as :ref:`experimental <software_maturity>`.
 
+.. note::
+   The LE Audio Controller Subsystem for nRF53 associated with this release comes with QDID 181316.
```

Comment:
How about putting this on the feature support page before the table include? And also on the bin library page?
Also, `this release` is unclear -- you mean NCS v2.4.0, NCS v2.4.1, v2.4.99, or the new version of the controller?
```suggestion
.. note::
   The :ref:`lib_bt_ll_acs_nrf53_readme` associated with this application comes with the QDID 181316.
```

Reply:
Changed to what we discussed on the Teams call